scripts/upload-secrets/upload-slack-secrets.rb

```diff
@@ -11,11 +11,14 @@
 credhub_namespaces = [
   "/concourse/main/create-cloudfoundry",
   "/#{deploy_env}/#{deploy_env}",
+  "/concourse/main/fast-startup-and-shutdown-cf-env",
```

Comment:
You are exposing both secrets to fast-startup-and-shutdown-cf-env. You should just expose what you need.

Reply:
fixed